src/mono/wasm/Wasm.Build.Tests/WasmTemplateTests.cs

```diff
@@ -185,15 +185,16 @@ public void ConsoleBuildThenPublish(string config)
         [InlineData("Release", false)]
         [InlineData("Release", true)]
         public void ConsoleBuildAndRunDefault(string config, bool relinking)
-            => ConsoleBuildAndRun(config, relinking, string.Empty);
+            => ConsoleBuildAndRun(config, relinking, string.Empty, DefaultTargetFramework);
 
         [ConditionalTheory(typeof(BuildTestBase), nameof(IsUsingWorkloads))]
-        [InlineData("Debug", "-f net7.0")]
-        [InlineData("Debug", "-f net8.0")]
-        public void ConsoleBuildAndRunForSpecificTFM(string config, string extraNewArgs)
-            => ConsoleBuildAndRun(config, false, extraNewArgs);
+        // [ActiveIssue("https://github.com/dotnet/runtime/issues/79313")]
```

Comment:
Probably forgotten

Reply:
Forgotten what? Once that PR is merged and we have packages available with the fix, this can be re-enabled.